### ISSUE TITLE
Admin API - Kafka 

### ISSUE THREAD

**Issue by jebrubaker** [confluentinc/confluent-kafka-dotnet#1626]
We would like to request the ability to manipulate Kafka ACL's through this .net interface

Per conversations, we understand the issue mentioned here to be a dependency.

https://github.com/edenhill/librdkafka/issues/2058

Thanks,
Jeff

**Comment by mhowlett** [comment]
no, not a near term priority. these are awkward to add to the bindings, and we are prioritizing great admin coverage in REST Proxy (this should be in the latest release) instead for the moment. Also, the REST API will be available directly in Confluent Cloud before too long, if it's not already.

**Comment by jebrubaker** [comment]
Thank you, to firm up this conversation we are understanding the recommended path is to proceed here. https://docs.confluent.io/platform/current/kafka-rest/api.html#rest-proxy-v3

**Comment by mhowlett** [comment]
yes

**Comment by mhowlett** [comment]
you could also consider calling out to the command line utils that come with kafka. that's not completely hassle/additional dependency free though - requires java installed.
